test/liquidate/reactivate.ts

```diff
@@ -98,6 +96,23 @@ describe('Reactivate Tests', () => {
     const updatedCluster = liquidatedCluster.eventsByName.ClusterLiquidated[0].args;
     await helpers.DB.ssvToken.connect(helpers.DB.owners[1]).approve(ssvNetworkContract.address, helpers.CONFIG.minimalOperatorFee);
 
-    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).reactivate(updatedCluster.operatorIds, helpers.CONFIG.minimalOperatorFee, updatedCluster.cluster)).to.be.revertedWithCustomError(ssvNetworkContract,'InsufficientBalance');
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).reactivate(updatedCluster.operatorIds, helpers.CONFIG.minimalOperatorFee, updatedCluster.cluster)).to.be.revertedWithCustomError(ssvNetworkContract, 'InsufficientBalance');
+  });
+
+  it('Reactivate a liquidated cluster after making a deposit', async () => {
+    await utils.progressBlocks(helpers.CONFIG.minimalBlocksBeforeLiquidation);
+    const liquidatedCluster = await trackGas(ssvNetworkContract.liquidate(firstCluster.owner, firstCluster.operatorIds, firstCluster.cluster));
+    let clusterData = liquidatedCluster.eventsByName.ClusterLiquidated[0].args;
+
+    await helpers.DB.ssvToken.connect(helpers.DB.owners[1]).approve(ssvNetworkContract.address, minDepositAmount);
+    const depositedCluster = await trackGas(ssvNetworkContract.connect(helpers.DB.owners[1]).deposit(
+      firstCluster.owner,
+      firstCluster.operatorIds,
+      minDepositAmount,
+      clusterData.cluster));
+    clusterData = depositedCluster.eventsByName.ClusterDeposited[0].args;
+
+    await helpers.DB.ssvToken.connect(helpers.DB.owners[1]).approve(ssvNetworkContract.address, minDepositAmount);
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).reactivate(firstCluster.operatorIds, minDepositAmount, clusterData.cluster)).to.emit(ssvNetworkContract, 'ClusterReactivated');
```

Comment:
Can you change minDepositAmount to 0?  We have tests for reactivate, im wondering if we deposit enough first then reactivate with 0 if it will work

Reply:
Yes, updated (and working)